mycroft/audio/cache_handler.py

```diff
@@ -0,0 +1,130 @@
+# Cache handler - reads all the .dialog files (The default
+# mycroft responses) and does a tts inference.
+# It then saves the .wav files to mark1 device
+
+import base64
+import glob
+import os
+import re
+import shutil
+import hashlib
+import json
+from urllib import parse
+from requests_futures.sessions import FuturesSession
+from mycroft.util.log import LOG
+
+
+REGEX_SPL_CHARS = re.compile('[@#$%^*()<>/\|}{~:]')
```

Comment:
You should use `r'[@#$%^*()<>/\|}{~:]'` (raw string literal) to make sure python doesn't interpret some of this as escaped characters